_includes/about.html

```diff
@@ -1,20 +1,21 @@
-<section class="bg-primary" id="about">
+<section class="bg-dark" id="about">
     <div class="container">
         <div class="row">
             <div class="col-lg-8 col-lg-offset-2 text-center">
-                <h2 class="section-heading">It's hard work to collaborate on floral designs.</h2>
-                <hr class="light">
-                <p class="text-faded">A picture is worth a thousand words. But for custom flower orders, that picture won’t exist until the flower arrangement has already been made. Until then, clients and florists have to find other ways to communicate their artistic vision: emails, phone calls, Pinterest boards. It’s a process that takes trial and error, especially when a client doesn’t know what they want.
+                <h2 class="section-heading">
+                    At Foxtail, we know that the best custom bouquets come from local florist shops.</h2>
+                <!-- <hr class="light"> -->
+                <p>
+                    But as customers move online, they spend more time on your website than your shop.
+                    Let's meet them where they are, and bring a delightful custom bouquet experience to your website. 
                 </p>
-                <p class="text-left">BEFORE FOXTAIL: </p>
-                <img src="img/before.png" class="img-responsive" alt="">
-                <p class="text-faded">
-                    At Foxtail, we turn floral design ideas into visual images early on in the design process. We’re building online tools to make it easier for clients and florists to understand what an order will look like, from color palettes to shape and style.
+                <h2>
+
+                <p>                    
+                    Foxtail enhances your website by empowering your customers to build their own bouquets online.
+                    Our creative tools for custom boquets are easy to add on to your website—no technology experience required.
```

Comment:
```suggestion
                    Our creative tools for custom bouquets are easy to add to your website--no technology experience required.
```

Reply:
this is actually an em dash, so it'll appear as a long dash. fixed the other typos